shared/scss/popup.scss

```diff
@@ -116,9 +116,37 @@ body {
             margin-left: 12px;
         }
 
+        .site-info__protection--is-whitelisted {
+            opacity: 100;
+            margin-top: 0;
+            transition: all 1.5s ease;
+            -webkit-transition: all 1.5s ease;
+        }
+
+        .site-info__protection--is-whitelisted.is-transparent {
+            margin-top: -16px;
+            opacity: 0;
+        }
+ 
+        .site-info__protection {
+            opacity: 100;
+            margin-top: 0;
+            max-height: 20px;
+        }
+
+        .site-info__protection.is-transparent {
+            margin-top: 16px;
+            opacity: 0;
+            max-height: 0;
+            transition: all 1.5s ease;
+            -webkit-transition: all 1.5s ease;
```

Comment:
Couple of nitpicks here, sorry:

- max opacity is `1`
- no need for the prefix on `-webkit-transition`, Chrome and Safari should support this unprefixed pretty well by now